### DISCUSSION TITLE
Possible to restore classic menus to pix?

### DISCUSSION THREAD

**Discussion by alameth** [linuxmint/discussions#165] [Questions]
I'm wondering what is the level of effort to restore the classic (non CSD) menus to pix? I'm disabled, pix is probably the most used app in my daily workflow, and I can just barely use it right now.

Happy to open an issue on pix itself, but as a n00b was hoping for some guidance from the community before I do so; when I write an issue I want it to be actionable.

**Comment by marco4mp** [comment]
In my case, I like it better as it looks now. I haven't had any major problems with the change.
I also like it because it makes better use of screen space.
